lib/smart_proxy_dns_infoblox/plugin_configuration.rb

```diff
@@ -0,0 +1,19 @@
+module Proxy::Dns::Infoblox
+  class PluginConfiguration
+    def load_classes
+      require 'infoblox'
+      require 'dns_common/dns_common'
+      require 'smart_proxy_dns_infoblox/dns_infoblox_main'
+    end
+
+    def load_dependency_injection_wirings(container_instance, settings)
+      container_instance.dependency :connection,
+                                    lambda {::Infoblox::Connection.new(:username => settings[:infoblox_user] ,:password => settings[:infoblox_pw], :host => settings[:infoblox_host])}
```

Comment:
It would be nice to unify the settings with the dhcp infoblox. Although, I guess there is a difference on the dns vs dhcp, that the dns doesn't have the server setting in the main config file. Anyway, we call it `server` on dhcp, while here it is `infoblox_host`, so at least `infoblox_server` would look more clear. Also, in `dhcp` we set the `ssl_opts` while here we leave it blank.